blazor/treegrid/rows/row-height.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Row Height in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Row Height in Syncfusion Blazor Tree Grid component and much more details.

```

Comment:
description: Check out here and learn more details about the Row Height in the Syncfusion Blazor Tree Grid component.